src/components/OfflineWithFeedback.js

```diff
@@ -73,13 +81,14 @@ function applyStrikeThrough(children) {
 }
 
 const OfflineWithFeedback = (props) => {
-    const hasErrors = !_.isEmpty(props.errors);
+    const hasErrors = props.hasErrors || !_.isEmpty(props.errors);
     const isOfflinePendingAction = props.network.isOffline && props.pendingAction;
     const isUpdateOrDeleteError = hasErrors && (props.pendingAction === 'delete' || props.pendingAction === 'update');
     const isAddError = hasErrors && props.pendingAction === 'add';
     const needsOpacity = (isOfflinePendingAction && !isUpdateOrDeleteError) || isAddError;
     const needsStrikeThrough = props.network.isOffline && props.pendingAction === 'delete';
     const hideChildren = !props.network.isOffline && props.pendingAction === 'delete' && !hasErrors;
+    const showErrorMessages = props.shouldShowErrorMessages && hasErrors && !_.isEmpty(props.errors);
```

Comment:
hasErrors already has that condition
```suggestion
    const showErrorMessages = props.shouldShowErrorMessages && hasErrors;
```